### PR TITLE
[TRANSLATION] Chinese Traditional translation improvements

### PR DESCRIPTION
**Pull request by AobiYT**
Creating another pull request before the last request is completed LOL

## Purpose
Add Chinese Traditional translation for [CHARMAP_NEW], [DRWTSN32], [UTILMAN] and [CHKDSK] and improve some of the translations for other files.

## Reason of changes
- consistency issues (eg. 禁用 > 停用, 高階 > 進階)
- attempt to unify the format of translation
- correct terms that used in Mainland China BUT not in Taiwan (also in HK and Macau)
- correcting some some sentence(s) that are not easy to understand
- untranslating some terms that I don't think they should be translated (eg. `IDS_UsageMessage "%s\n\n用法：regsvr32 [/u] [/s] [/c] [/n] [/i[:命令列]] DLL名稱\n\` > `IDS_UsageMessage "%s\n\n用法：regsvr32 [/u] [/s] [/c] [/n] [/i[:cmdline]] dllname\n\` )

## Caution
The file `winhlp32` is come from Wine, so I have decided to import most of the translation from Wine to the file. The translation can be found on [here](https://source.winehq.org/git/wine.git/blob/13ea90d80f7275e1ad4f3fc3c1c75b68bdbefbb4:/po/zh_TW.po).

## TODO (in later pull requests)
- improve some of the translations for other files (still alot)
- planning to add Chinese Hong Kong (zh-HK) language support

### CODE REVIEW

**Comment by AobiYT** [comment]
but those terms are not translated in Windows either, still translate them?

**Comment by HBelusca** [comment]
> but those terms are not translated in Windows either, still translate them?

Regarding translations, we can always improve what Windows has. So for example, you personally think that it would be nice to translate these "cmdline" terms etc. in the help, while Windows didn't do that for unknown reasons (Note for example, that in french these terms are translated), then just go ahead and translate them!

**Comment by AobiYT** [comment]
ok translated

**Comment by binarymaster** [comment]
Cc @reactos/lang-chinese, and chinese speaking people @chirsz-ever, @njlyf2011, @laosb, @Frozen-Tofu for review / approvals

**Comment by Frozen-Tofu** [comment]
lgtm

File: base/applications/drwtsn32/lang/zh-TW.rc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*
+ * PROJECT:     Dr. Watson crash reporter
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     zh-TW resource file
+ * TRANSLATOR:  Aobi Chan CL (eason066 [at] gmail [dot] com), 2020
+ * COPYRIGHT:   Copyright 2018 William Kent (wjk011 [at] gmail [dot] com)
```
Comment (HBelusca) [inline base/applications/drwtsn32/lang/zh-TW.rc:6]:
```suggestion
 * COPYRIGHT:   Aobi Chan CL (eason066 [at] gmail [dot] com), 2020
```

File: base/applications/winhlp32/lang/Zh.rc [hunk 2]
```diff
@@ -129,67 +130,67 @@ MAIN_MENU MENU
...
+			MENUITEM "中",      MNID_CTXT_FONTS_NORMAL
+			MENUITEM "大",       MNID_CTXT_FONTS_LARGE
 		END
 	END
 END
 
 STRINGTABLE
 {
-STID_WINE_HELP, 	"Wine 幫助"
+STID_WINE_HELP, 	"ReactOS 說明"
```
Comment (HBelusca) [inline base/applications/winhlp32/lang/Zh.rc:166]: :D

**Pull request merged**